RELEASING.md

```diff
@@ -0,0 +1,13 @@
+## To generate a release of a tool
+
+```
+./release.sh <TOOL_NAME>
+```
+
+this command will generate files to release under the corresponding <TOOL_NAME> folder, e.g.
+
+```
+./release.sh dp_check
+```
+
+generates dp_check binary under dp_check/, then you can create a release using those generated files.
```

Comment:
can we add details here for the specific steps taken to publish the release.

I.e., something along the lines of
a) "create new git tag with ..."
b) "publish a new release with ...."